core/templates/domain/exploration/SolutionObjectFactory.ts

```diff
@@ -105,12 +104,21 @@ export class Solution {
         new UnitsObjectFactory())).fromDict(
           this.correctAnswer as NumberWithUnitsAnswer).toString();
     } else if (interactionId === 'DragAndDropSortInput') {
-      const formatRtePreview = new FormatRtePreviewPipe(new CapitalizePipe());
+      if (!customizationArgs) {
+        throw new Error(
+          'Customization args for the drag and drop interaction are missing');
+      }
       correctAnswer = [];
+      let subtitledHtmlChoices = (
+        customizationArgs as DragAndDropSortInputCustomizationArgs)
+        .choices.value;
+      let subtitledHtmlChoicesContentIds = subtitledHtmlChoices.map(
+        choice => choice._contentId);
```

Comment:
Prefer using the getter instead of accessing the private variables.

Reply:
Got it